docs/pages/admin-guides/management/operations/breakglass-access.mdx

```diff
@@ -0,0 +1,197 @@
+---
+title: Configuring Break Glass Access for Disaster Recovery
+description: Guide to set up break glass emergency access for critical systems if Teleport becomes unavailable.
+---
+
+This guide will walk you through the steps required to configure emergency "break glass" access for critical agents and servers via OpenSSH using Teleport-issued certificates, in the following scenarios:
+
+1. The Teleport Agent on a server has crashed, gone offline, or become unusable.
+2. The Teleport control plane is down and cannot be used to access systems, and out-of-band access is necessary to fix it.
+
+## How it works
+
+Teleport's CA can issue short-lived, signed certificates that can be used to grant access to OpenSSH servers in emergency disaster recovery scenarios. By configuring the OpenSSH server on the Teleport Agent to trust Teleport’s CA, users with valid Teleport-issued certificates can authenticate to the server without requiring static SSH keys or passwords even if Teleport itself is down.
+
+Below, we’ll detail the steps to accomplish the following objectives:
+
+1. Configure the Teleport Agent's OpenSSH server to trust Teleport's CA
+2. Create a `breakglass` system user on the remote host
+3. Restrict cert-based SSH access to this `breakglass` user only
+4. Create a `breakglass` Role and User in Teleport
+5. Generate `breakglass` SSH Key and Cert using Teleport's CA
+6. Access the remote server using a Teleport issued cert even if Teleport is down or unresponsive
+
+## Prerequisites
+
+- OpenSSH client version 7.4 or above on your local machine.
+- A Linux host with OpenSSH server (`sshd`) version 7.4 or above.
+
+(!docs/pages/includes/commercial-prereqs-tabs.mdx!)
+- A Teleport user with ability to add new `impersonate` role permissions.
+
+## Step 1/6. Configure sshd to trust the Teleport CA
+
+For break glass access, the OpenSSH server must be configured to trust client certificates issued by the Teleport Certificate Authority (CA).
+
+1. **Export the Teleport CA public key:**
+
+On the OpenSSH server, run the following command. Replace `proxy.example.com` with the address of your Teleport Proxy Service:
+
+```bash
+export KEY=$(curl 'https://proxy.example.com/webapi/auth/export?type=openssh' | sed "s/cert-authority\ //")
```

Comment:
The `openssh` CA is not the correct issuer to export, certificates signed by the `openssh` are never exposed to users and are only ever kept in memory in the proxy.